lib/commands/create/function.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const promptFunctionName = require('../../prompts/function-name');
+const path = require('../../prompts/path');
+
+const functionWithPayloadSchemaTemplate = require('../../templates/function-with-payload-schema');
+const functionWithoutPayloadSchemaTemplate = require('../../templates/function-without-payload-schema');
+const functionSchemaFs = require('../../fs/function-schema');
+
+const functionSourceTemplate = require('../../templates/function-source');
+const functionSourceFs = require('../../fs/function-source');
+
+const functionTestTemplate = require('../../templates/function-test');
+const functionTestFs = require('../../fs/function-test');
+
+const { ensureOptions } = require('../../utils');
+const { Report } = require('../../report');
+
+
+const getPrompts = () => [
+
+	promptFunctionName,
+	{ ...path, message: 'What\'s the function path?', initial: './src/lambda/{functionName}.js' },
```

Comment:
`{ ...path, message: 'What\'s the function path?', initial: functionName => `./src/lambda/${functionName}.js`

Yo pondria esto en el prompt. De esta manera si vos pones `function-test`, el path mostrado va a ser `./src/lambda/FunctionTest.js`